backend/frontend/static/frontend/user_board.js

```diff
@@ -21,3 +21,51 @@ function moveCardToCorrectColumn(cardId, column) {
     destinationColumn.insertBefore(card, destinationColumn.firstChild);
   }
 }
+
+
+/*
+This inserts an alert above the card being acted on.
+It's used to show any errors or success messages that 
+come back from the server.
+*/
+function showCardAlert({ cardId, message, alertType = "info", persist = true }) {
```

Comment:
This is similar to some code from Ngoako that I just reviewed.  You shouldn't be implementing the same thing in 2 different PRs.